module/User/src/Model/User.php

```diff
@@ -157,7 +158,7 @@ public function getRoleNames()
     public function getRoleId(): string
     {
         $roleNames = $this->getRoleNames();
-        if (in_array('admin', $roleNames)) {
+        if (in_array('admin', $roleNames) || $this->getMember()->isBoardMember()) {
             return 'admin';
         }
```

Comment:
This is essentially what I suggested in #1147, just without the new `board` role (which I personally really like) and the check in `Member` instead of `User`.

I suggest we await some more feedback, but otherwise this seems like a good solution.

Reply:
Since board and admin permissions are not different, I propose not having seperate roles.
The installations required for checking wheteher someone is a board member are all located in the decision module. So, the check should be there too.